docs/guide/start/blueprints.md

```diff
@@ -45,17 +45,14 @@ application. Your application will be shown as "Starting" on the web console's f
 
 Instead of pasting the YAML blueprint each time,
 this blueprint can be [added to the catalog](../ops/catalog/).
-Or, even easier, you can download a sample [catalog.xml](catalog.xml).
-Install this to your `~/.brooklyn/` folder and relaunch Brooklyn
-(navigating to the "Help" tab in order to shutdown Brooklyn *and* the application you launched in the previous step).
-
-Now when the web console is re-opened, the catalog contains our blueprints.
-Select the "Demo Web Cluster with DB" and click "Next".
+With this YAML blueprint added, including the location, the Add Application dialog will offer 
```

Comment:
Is this the only mention in the docs that "including the location" will treat the catalog item as a template? Deserves a section of its own in `docs/guide/ops/catalog/index.md`, including how to add the catalog item as a "tile" (i.e. as a clickable icon that will not show the yaml, but will instead allow it to be deployed directly - a customer was recently stressing the importance of that when they demo Brooklyn to their own customers).